tests/installation/select_patterns.pm

```diff
@@ -29,4 +29,17 @@ sub run {
     $self->accept_changes();
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    my $file_tar = "/tmp/y2logs_select_patterns.tar.bz2";
+
+    select_console 'log-console';
+    script_run "save_y2logs $file_tar";
```

Comment:
This should just be

```
sub post_fail_hook {
    my $self = shift;

    select_console 'log-console';
    my $ret = script_output("grep -E -m 1 \"nothing provides\" /var/log/YaST2/y2log", proceed_on_failure => 1);
    record_info("Conflict:", $ret, result => 'fail') if ($ret);
    $self->SUPER::post_fail_hook();
}
```

Or is there some other reason to first tar the logs and immediately extract them again?

Reply:
Yeah, I missed it in this way of doing. Thanks for suggesting, and I have modified the code.